src/Scripts/DefaultVirtualScrips.spec.ts

```diff
@@ -24,10 +23,10 @@ describe('DefaultVirtualScripts', () => {
         VirtualScript
       ][] = [...virtualScripts];
       // assert
```

Comment:
There are a few empty lines were missed like the following:
```suggestion
      virtualScripts.set(key2, VirtualScriptType.REMOTE, 'let a = 2;');
      
      // act
      // eslint-disable-next-line @typescript-eslint/typedef
      const [[firstKey, firstScript], [secondKey, secondScript]]: [
        string,
        VirtualScript
      ][] = [...virtualScripts];
      
      // assert
      expect(firstKey).toBe(key1);
```

Reply:
The same as in the `VirtualScript.spec.ts` and in the other files.